etcdctl/ctlv3/command/printer_json.go

```diff
@@ -42,6 +42,50 @@ func (p *jsonPrinter) EndpointStatus(r []epStatus) { printJSON(r) }
 func (p *jsonPrinter) EndpointHashKV(r []epHashKV) { printJSON(r) }
 func (p *jsonPrinter) DBStatus(r snapshot.Status)  { printJSON(r) }
 
+func (p *jsonPrinter) Get(r clientv3.GetResponse) {
+	if !p.isHex {
+		printGetWithHexJSON(r)
+	} else {
+		printJSON(r)
+	}
+}
+
+func printGetWithHexJSON(r clientv3.GetResponse) {
```

Comment:
I see that we already have this pattern, but hand-concatenating JSON seems to me to be code-small. 
Can we map this structure to another struct with proper types/json annotations and use the regular marshaller  instead ?

Reply:
@ptabor Yeah. The hand-coding did not seem to be right but since there was a precedent, I did it that way. Is there an example of json annotations etc. that you can point to?

Reply:
I'm afraid you would need to have a sibling struct (or proto), that instead of: 

```
type KeyValue struct {
	...
	Value []byte `protobuf:"bytes,5,opt,name=value,proto3" json:"value,omitempty"`
}
```

we would need: 

type JsonKeyValue struct {
	...
	Value string  json:"value,omitempty"`
}

In general the conversion to string starts to be tricky if the bytes contains not correct UTF-8 sequence.